src/button.js

```diff
@@ -1,7 +1,7 @@
 import React, {Component} from 'react'
 import PropTypes from 'prop-types'
 
-import {AUTOCOMPLETE_CONTEXT} from './constants'
+import AUTOCOMPLETE_CONTEXT from './context'
```

Comment:
Sounds good 👍 I kept getting an eslint warning because it wanted it to be a single default export so I changed it to one export. I'll put it back though. Should I just add an ignore comment or something?